docs/build/apps/dapp-frontend.mdx

```diff
@@ -1,561 +1,536 @@
 
-Notice that we were able to use the contract alias, `hello_world`, in place of the contract id!
+If none of them suit, then it might be time to...
 
-:::
+## Make your own template
 
-This project is set up as an NPM Workspace, and so the `hello_world` client library was generated in the `packages` directory at `packages/hello_world`.
+Let’s make our own template! In this example template, we use SolidJS as the JavaScript framework, but other frameworks can be used with minor modifications. The template is using the `hello world` example smart contract, and is a part of the template initialization; bindings for the `hello world` smart contract are created.
 
-We attempt to keep the code in these generated libraries readable, so go ahead and look around. Open up the new `packages/hello_world` directory in your editor. If you've built or contributed to Node projects, it will all look familiar. You'll see a `package.json` file, a `src` directory, a `tsconfig.json`, and even a README.
+This example template is very simple, most of the work goes into creating the `initialize.js` file, which is used to take care of creating a user account, building and deploying the smart contract, and creating the smart contract TypeScript bindings.
 
-## Generate an NPM package for the Increment contract
+### 1. Initialize a SolidJS project
 
-Though we can run `stellar contract bindings typescript` for each of our contracts individually, the [soroban-template-astro](https://github.com/stellar/soroban-astro-template) project that we used as our template includes a very handy `initialize.js` script that will handle this for all of the contracts in our `contracts` directory.
+```bash
 
-In addition to generating the NPM packages, `initialize.js` will also:
+npx degit solidjs/templates/ts soroban-template-solid
+cd soroban-template-solid
+npm install
+npm run dev
 
-- Generate and fund our Stellar account
-- Build all of the contracts in the `contracts` dir
-- Deploy our contracts
-- Create handy contract clients for each contract
+```
 
-We have already taken care of the first three bullet points in earlier steps of this tutorial, so those tasks will be noops when we run `initialize.js`.
+The basic SolidJS is now running on localhost port 3000.
 
-### Configure initialize.js
+#### Dependencies
 
-We need to make sure that `initialize.js` has all of the environment variables it needs before we do anything else. Copy the `.env.example` file over to `.env`. The environment variables set in `.env` are used by the `initialize.js` script.
+Most of the needed dependencies are already included by the SolidJS template, we just need to add three more:
 
 ```bash
-cp .env.example .env
-```
 
-Let's take a look at the contents of the `.env` file:
+npm install dotenv glob util
 
```

Comment:
I think you might have some extra line-breaks surrounding some of the bash code blocks. make sure there are no blank lines between the three backticks on either side of the code